prisma/schema.prisma

```diff
@@ -18,4 +18,45 @@ datasource db {
 model Survey {
   id         Int       @id @default(autoincrement())
   name       String
-}
\ No newline at end of file
+
+  // Relationship info for Prisma client - NOT database columns
+  questions  Question[]
+
+  @@map("survey")
+}
+
+
+model Question {
+  id         Int     @id @default(autoincrement())
+  name       String
+
+  // Columns that will be used to store Relationship key
+  surveyId   Int
+
+  // Relationship info for Prisma client - NOT database columns
+  surveys    Survey @relation(fields: [surveyId], references: [id])
+  answers    Answer[]
+
+  @@map("question")
+}
+
+
+
+model Answer {
+  id         Int     @id @default(autoincrement())
+  name       String
+
+  // Columns that will be used to store Relationship key
+  questionId Int
+
+  // Relationship info for Prisma client - NOT database columns
+  questions  Question @relation(fields: [questionId], references: [id])
```

Comment:
Should this column be called "Question" rather than "Questions"? Its a singular question the given answer is associated with. Similar for "Surveys" column on Question model above